### PR TITLE
DOCS-1068: Add full image view

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/manage/data/label.md [hunk 1]
```diff
@@ -57,6 +61,8 @@ Select the **Image Tags** option.
 
 {{<gif webm_src="/manage/data/tag-star.webm" mp4_src="/manage/data/tag-star.mp4" alt="Tag image with a star label">}}
 
+If you want to see the image in bigger, click on the expand side menu arrow.
```
Comment (JessamyT) [inline docs/manage/data/label.md:64]:
```suggestion
If you want to see the image bigger, click on the expand side menu arrow.
```
or
```suggestion
If you want to expand the image, click on the expand side menu arrow.
```

File: docs/manage/data/label.md [hunk 2]
```diff
@@ -38,6 +38,10 @@ Click on the image where you would like to add the bounding box and drag to wher
 
 {{<gif webm_src="/manage/data/label-dog.webm" mp4_src="/manage/data/label-dog.mp4" alt="Add a bounding box around the dog in an image">}}
 
+If you want to see the image in bigger, click on the expand side menu arrow:
```
Comment (JessamyT) [inline docs/manage/data/label.md:41]: I think "in bigger" sounds wrong
```suggestion
If you want to see the image bigger, click on the expand side menu arrow.
```
or
```suggestion
If you want to expand the image, click on the expand side menu arrow.
```
maybe both this and the other one should either have a colon or a period?

Comment (npentrel) [reply docs/manage/data/label.md:41]:
```suggestion
If you want to expand the image, click on the expand side menu arrow:
```

**Review by JessamyT — APPROVED** [review]
LGTM unless you want to actually indicate where the "expand side menu arrow" is verbally or with a red box on the GIFs or something

File: docs/manage/data/label.md [hunk 3]
```diff
@@ -38,6 +38,10 @@ Click on the image where you would like to add the bounding box and drag to wher
 
 {{<gif webm_src="/manage/data/label-dog.webm" mp4_src="/manage/data/label-dog.mp4" alt="Add a bounding box around the dog in an image">}}
 
+If you want to expand the image, click on the expand side menu arrow:
```
Comment (npentrel) [inline docs/manage/data/label.md:41]:
```suggestion
If you want to expand the image, click on the expand side menu arrow in the corner of the image:
```

File: docs/manage/data/label.md [hunk 4]
```diff
@@ -57,6 +61,8 @@ Select the **Image Tags** option.
 
 {{<gif webm_src="/manage/data/tag-star.webm" mp4_src="/manage/data/tag-star.mp4" alt="Tag image with a star label">}}
 
+If you want to see the image bigger, click on the expand side menu arrow.
```
Comment (npentrel) [inline docs/manage/data/label.md:64]:
```suggestion
If you want to expand the image, click on the expand side menu arrow in the corner of the image.
```

**Pull request merged**